src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.ts

```diff
@@ -49,6 +52,36 @@ export class InstructorHelpQuestionsSectionComponent extends InstructorHelpSecti
     showRecipientNameTo: [FeedbackVisibilityType.INSTRUCTORS, FeedbackVisibilityType.RECIPIENT],
   };
 
+  readonly exampleNumericalScaleEditFormModel: QuestionEditFormModel = {
+    feedbackQuestionId: 'CS3281',
+
+    questionNumber: 1,
+    questionBrief: 'This is a brief of the question',
+    questionDescription: 'This is the description of the question',
+
+    isQuestionHasResponses: false,
+
+    questionType: FeedbackQuestionType.NUMSCALE,
+    questionDetails: DEFAULT_NUMSCALE_QUESTION_DETAILS(),
+
+    giverType: FeedbackParticipantType.STUDENTS,
+    recipientType: FeedbackParticipantType.OWN_TEAM_MEMBERS,
+
+    numberOfEntitiesToGiveFeedbackToSetting: NumberOfEntitiesToGiveFeedbackToSetting.CUSTOM,
+    customNumberOfEntitiesToGiveFeedbackTo: 1,
+
+    showResponsesTo: [],
+    showGiverNameTo: [],
+    showRecipientNameTo: [],
```

Comment:
Would be good to follow the defaults created for a new question https://github.com/TEAMMATES/teammates/blob/331faaa71954c7530125aca3fefebca6d27c2837/src/web/services/feedback-questions.service.ts#L332-L348

Reply:
I think it is even better to use the service to generate a template and them fill dummy values such as `questionBrief` or `questionDescription`.

Reply:
> I think it is even better to use the service to generate a template and them fill dummy values such as `questionBrief` or `questionDescription`.

You mean create a new method in the service specifically for populating help page data? I like the idea but feel that it's something we can work on after v7?

Reply:
It's fine. I am thinking about something else. Just ignore my comment :P